### PR TITLE
config: add ti-sync-bot config

### PR DESCRIPTION
**Pull request by Mini256**

### CODE REVIEW

File: ti-community-bots/Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+SHELL := /bin/bash
+
+bots-namespace:
+	kubectl apply -f cluster/bots_namespace.yaml
+
+ti-sync-bot-config:
+	kubectl apply -f config/ti_sync_bot_config.yaml -n ti-community-bots
```
Comment (hi-rustin) [inline ti-community-bots/Makefile:7]: Please put namespace in yaml and not in the command.

File: ti-community-bots/config/database_config.yaml.example [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (hi-rustin) [inline ti-community-bots/config/database_config.yaml.example:1]: It should be example.yaml not yaml.example.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: ti-community-bots/config/database_config.yaml.example (hunk 2 above)
Comment (Mini256) [reply ti-community-bots/config/database_config.yaml.example:1]: This is to prevent the example file from being applied.

Comment (hi-rustin) [reply ti-community-bots/config/database_config.yaml.example:1]: It's not readable. Maybe we can move it into examples.

**Review by hi-rustin — APPROVED** [review]

**Comment by hi-rustin** [comment]
Better move it into dev directory.

/ti-community-bots/dev or /ti-community-bots-dev?

**Pull request merged**